### DISCUSSION TITLE
Direct entry of the impacts

### DISCUSSION THREAD

**Discussion by PBD-ILR-LU** [monarc-project/MonarcAppFO#502] [General]
Monarc users frequently face difficulties when needing to enter the impact levels, because they don’t know how to access them.
It would be more intuitive if the impact table could be directly accessed by clicking on the impact cells.
Thanks

**Comment by ruslanbaidan** [comment]
Holding mouse cursor on one of the consequences names' headers for around 3 seconds would display a popup with the information.
The same is applicable to the risks table - holding mouse cursor on risks probability column or vulnerability qualification would display the scales information.
Attached screenshots.
![Screenshot 2023-10-05 at 09 59 22](https://github.com/monarc-project/MonarcAppFO/assets/3246171/b917e883-7380-4fbd-bf69-60efb09fa764)
![Screenshot 2023-10-05 at 09 58 53](https://github.com/monarc-project/MonarcAppFO/assets/3246171/71042905-047f-4495-8653-737329956025)

**Comment by ruslanbaidan** [comment]
Based on the fact that it is possible to set impacts only for all the linked risks linked to asset at the same time.
As a solution of direct impact setup in the table view could be - opening the same dialog of impacts edit and set the values for the linked asset's risks, where mouse was clicked.
There could be added a specific message informing about the asset for which the impacts are set and selecting it in the analysis tree.
However, there could be some concerns regarding risks linked to a global assets that are presented in the analysis more then once.

Created issue based on the discussion: https://github.com/monarc-project/MonarcAppFO/issues/521
